src/UI/Factory.php

```diff
@@ -662,4 +662,36 @@ public function breadcrumbs(array $crumbs);
 	 */
 	public function chart();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A Floatable overlays all other content on the screen. It's free floating
+	 *     nature allows the positioning to some fitting space on the screen. They may
```

Comment:
Does this mean, the user (i.e. person sitting in front of the screen here) is allowed to position it to its liking?